doc/rose-rug-advanced-tutorials-suicide.html

```diff
@@ -260,6 +260,282 @@ <h3 class="alwayshidden">Note on suicide triggers (2)</h3>
+
+    [[bar]]
+        inherit = FAMILY_PASS
+        command scripting = echo 'bar always succeeds'
+
+    [[dependent_on_families]]
+       command scripting = """
+           sleep 10;
+           echo 'I can only run if all FAMILY_PASS succeed and SOME_DO_SOME_DONT finish'
+           """
+
+    [[foo]]
+        inherit = FAMILY_PASS
+        command scripting = echo 'foo always succeeds'
+                           
+    [[flaky_member]]
+        inherit = SOME_DO_SOME_DONT
+        command scripting = """
+            echo 'flaky member is going to:'
+            sleep 10;
+            if (($RANDOM % 2)); then
+                echo 'Success'; true; 
+            else
+                echo 'Fail'; false;
+            fi
+        """
+    [[sometimes_fail]]
+       command scripting = """
+           sleep 10;
+           if (($RANDOM % 2)); then
+               echo 'Success'; true;
+           else
+               echo 'Fail'; false;
+           fi
+       """
+
+    [[stable_member]]
+        inherit = SOME_DO_SOME_DONT
+        command scripting = echo 'stable member always succeeds'
+</pre>
+    </div>
+
+      <p>and a <samp>suite-twofail.rc</samp> file that looks like this:</p>
+
+    <div class="slide">
+        <h3 class="alwayshidden">Advanced example suite-twofail.rc</h3>
+<pre class="prettyprint lang-cylc">
+# If the first task fails got the end of the cycle or if two specified task fail
+# go to the second from last task and then to the end of that cycle.
+# suite-twofail.rc
+
+[scheduling]
+    [[dependencies]]
+         graph = """
+             start_install          => check_files_exist
+             check_files_exist      => generate_plots
+             check_files_exist:fail => !generate_plots & !move_data
+             check_files_exist:fail | move_data => housekeep
+
+             generate_plots:fail | check_files_exist:fail => !raise_alert
+             generate_plots:fail | raise_alert => move_data
+             generate_plots         => raise_alert
+         """
+
+[runtime]
+    [[check_files_exist]]
+        command scripting = """
+            echo 'Do the files exist?'
+            sleep 10;
+            if (($RANDOM % 2)); then
+                echo 'Yes, then success'; true; 
+            else
+                echo 'No, then fail'; false;
+            fi
+        """
+
+    [[generate_plots]]
+        command scripting = """
+            echo 'You could run a script to plot data, but did they finish?'
+            sleep 10;
+            if (($RANDOM % 2)); then
+                echo 'Yes, then success'; true; 
+            else
+                echo 'No, then fail'; false;
+            fi
+        """
+
+    [[move_data]]
+        command scripting = """
+            sleep 10;
+            echo 'You could run a script to move data'
+        """
+    [[raise_alert]]
+        command scripting = """
+            sleep 10;
+            echo 'You need to raise an alert: ALERT!'
+        """
+</pre>
+    </div>
+
```

Comment:
I think this is a good way of getting the user set up and ready to go. To reinforce what you've said I think it'd then be good to go through running each example and what they should see and why (nothing in depth but for the people struggling with these concepts I think they'll need a bit more of a jumping on point/explanation).

For example you could do something like:
---
Example 1

The first example demonstrates setting a suite up so that if a member of a family fails it carries on. If a specified task fails, in this case `flaky_activity`, go to the end of that cycle and remove the failed task from from the task pool.

Edit your `rose-suite.conf` file so that `EXAMPLE="1"`, save your change and run the suite with `rose suite-run`.

If `flaky_activity` fails you should see ...

---

That in mind it may prove simpler to just create 3 separate suite.rc files, one for each example, and not bother with the include and swapping of jinja2 variable values (I can see the process that led to it being done that way, I just think it'd be simpler for purposes of examples/tutorial to reduce complexity of anything around the core message about suicide triggering).